src/tests/resistance/fuzzing.rs

```diff
@@ -2392,3 +2154,23 @@ fn encode_messages_and_corrupt_body_length_field(
         })
         .collect()
 }
+
+/// Returns the set of messages used for fuzz-testing.
+/// This notably excludes [Message::Version] because it is
+/// usually tested separately.
+fn default_fuzz_messages() -> Vec<Message> {
```

Comment:
We may want to add variants which aren't just `empty()`, but for now this should suffice.